cmd/lifecycle/main.go

```diff
@@ -26,11 +27,11 @@ func main() {
 	case "builder":
 		cmd.Run(&buildCmd{}, false)
 	case "exporter":
-		cmd.Run(&exportCmd{}, false)
+		cmd.Run(&exportCmd{exportArgs: exportArgs{platformAPI: platformAPI}}, false)
```

Comment:
Do the other stages not need `platformAPI`?

Reply:
They all will eventually, but I figured I would wait to inject into each stage when it is needed.